lms/djangoapps/program_enrollments/models.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
+Django model specifications for the Program Enrollments API
+"""
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils.translation import ugettext as _
+from model_utils.models import TimeStampedModel
+from simple_history.models import HistoricalRecords
+
+
+class ProgramEnrollments(TimeStampedModel):  # pylint: disable=model-missing-unicode
```

Comment:
Model names should be singular -- i.e., `ProgramEnrollment`.

Reply:
Oh thanks, my bad.